src/views/Advertisement/Advertisement.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import {
+  customPropTypes,
+  getElementType,
+  META,
+} from '../../lib'
+
+/**
+*An ad displays third-party promotional content
+*/
+function Advertisement(props) {
+  const {
+    className,
+  } = props
+
+  const classes = cx(
+    'ui',
+    'ad',
+    className
+  )
+  const ElementType = getElementType(Advertisement, props)
+
+  return <ElementType className={classes} />
```

Comment:
I added the meta property here: https://github.com/tarang9211/Semantic-UI-React/blob/5e34b354fcbd0a9da6847893c96172bac4564dee/src/views/Advertisement/Advertisement.js#L27 I was referring to the Feed component and structuring it that way. I'll push a new commit with your changes.